bindings/python/FloatingBaseEstimators/src/LeggedOdometry.cpp

```diff
@@ -58,11 +58,11 @@ void CreateLeggedOdometry(pybind11::module& module)
              py::arg("encoder_speeds"))
         .def("advance", &LeggedOdometry::advance)
         .def("reset_estimator",
-             py::overload_cast<const InternalState&>(&LeggedOdometry::resetEstimator),
+             py::overload_cast<const InternalState&>(&FloatingBaseEstimator::resetEstimator),
              py::arg("new_state"))
         .def("reset_estimator",
              py::overload_cast<const Eigen::Quaterniond&, const Eigen::Vector3d&>(
-                 &LeggedOdometry::resetEstimator),
+                 &FloatingBaseEstimator::resetEstimator),
              py::arg("new_imu_orientation"),
              py::arg("new_imu_position"))
```

Comment:
This is a leftover from https://github.com/dic-iit/bipedal-locomotion-framework/commit/b664420ffe03c1f623e64be6c0f97570db452073#diff-6b6bd58b2e396c9de52e1e2ac47eae19645f350a0d24cc7a9130864764f8e73d where the methods in LeggedOdometry was removed to avoid unnecessary confusion with those in the Base class, since base class expected base link pose to reset estimator, while legged odometry used imu pose.

So it would be helpful to modify,
```suggestion
                 &FloatingBaseEstimator::resetEstimator),
             py::arg("new_base_orientation"),
             py::arg("new_base_position"))
```

Reply:
Done by @traversaro  in https://github.com/dic-iit/bipedal-locomotion-framework/pull/380/commits/6fe2416670d68190a2e0ba91414a96182fc84b03!